src/elements/breadcrumbs/src/index.tsx

```diff
@@ -15,9 +15,7 @@ interface HomeIconProps extends React.HTMLAttributes<HTMLDivElement> {
 }
 
 const HomeIcon: React.FC<HomeIconProps> = ({ variant = 'base' }) => {
-  const { theme }: any = useThemeUI()
-
-  const iconColor = get(theme, `${lookup(variant)}.homeIcon.color`)
```

Comment:
What if the palette is not defined? Should we be using both of these options? Palette or iconColor?